.github/CONTRIBUTING.md

```diff
@@ -7,30 +7,30 @@ We would love your contribution, but please keep the following in account while
 
 If you are adding a YouTube Channel to the list, the channel must classify under the following
 
-- Atleast 100K subscribers on YouTube (except for some exceptions, on the discretion of KendallDoesCoding).
+- At least 100K subscribers on YouTube (except for some exceptions, at the discretion of KendallDoesCoding).
 - Uploaded a video or live-streamed to YouTube within the last 1 MONTH.
-- The channel is a individual and not a COMPANY (for eg: you cannot add Good Mythical Morning, T-Series, as they are companies and not individuals).
-- The channel should not be contriversial (certain exceptions can be made, up to the judgement of @KendallDoesCoding).
+- The channel should be an individual and not a COMPANY (for eg: you cannot add Good Mythical Morning, or T-Series, as they are companies and not individuals).
+- The channel should not be controversial (certain exceptions can be made, up to the judgement of @KendallDoesCoding).
 - Birthday Format must be as follows: D/M, YY (eg: 2 May, 2000)
   **- The final decision on whether the channel gets added or not, is up to the discretion of KendallDoesCoding. **
 
 # How to add a channel
 
-To add you would to need to follow two steps:-
-1. - Go to the website and visit "/addYoutuber" route or "<_website>/addYoutuber"
-   - Here fill all the details of the youtuber like Category, Channel Name, Birthday, Total Channel Views, Channel Link and click on add
-   - 
-   <img src="https://res.cloudinary.com/dwuyp1nss/image/upload/v1690489952/Website%20Demo/addYoutuber_elcoaz.png" alt="add youtuber" />
+To add a YouTuber, you would need to follow two steps:-
 
-2. - Now after adding youtuber's info on website, you have to update the README (Important as only it will mark as original contribution on GitHub)
-   - For this set up the repo in local and update README to add youtuber info
-   -
-   <img src="https://res.cloudinary.com/dwuyp1nss/image/upload/v1690489952/Website%20Demo/README_jrxz1a.png" alt="README.md" />
+1. - Go to the website and visit the "/addYoutuber" route.
```

Comment:
Instead of telling the user to go to the /addYouTuber route, I think it'd be better if we link it to them here: maybe something like.

```suggestion
1. - Go to the [add YouTuber page](http://youtubers-birthdays.kendalldoescoding.tech/addYouTuber) on the website.
```

Reply:
Let's also include this below this line.

Note: It may take a while to load the first time as it has to load the database code, etc.